cmd/contour/serve.go

```diff
@@ -29,6 +29,7 @@ import (
 	contourv1 "github.com/projectcontour/contour/apis/projectcontour/v1"
 	"github.com/projectcontour/contour/internal/annotation"
 	"github.com/projectcontour/contour/internal/contour"
+	contourv2 "github.com/projectcontour/contour/internal/contour/v2"
```

Comment:
it's a little unfortunate that `contourv1` refers to the API package, while `contourv2` refers to the internal package. We could use `contourv1api` or similar for the former. Not blocking.

Reply:
Yup I agree. =)

Reply:
I opened #2938.